cucumber/api/features/step_definitions/audit_steps.rb

```diff
@@ -1,10 +1,16 @@
 # frozen_string_literal: true
 
+require_relative '../support/logs_helpers'
+
+Given(/^I save my place in the audit log file for remote$/) do
+  save_num_log_lines unless Utils.local_conjur_server
+end
```

Comment:
can we use the step from `logs_steps` instead and load it under this profile? we can add there a check on `audit` and move `save_num_log_lines unless Utils.local_conjur_server` there if it exiss

Reply:
We can but I thought that since the step `there is an audit record matching` might need this step to precede it, it's clearer to have them both in the same step definition file.

Reply:
it doesn't really matter so i'm ok with your decision